API/Controllers/ProjectController.cs

```diff
@@ -226,5 +229,33 @@ public async Task<IActionResult> DeleteProject(int projectId)
             projectService.Save();
             return Ok();
         }
+
+
+        /// <summary>
+        /// Gets the wizard information.
+        /// </summary>
+        /// <param name="sourceUrl">The source URL.</param>
+        /// <returns></returns>
+        [HttpGet("wizard")]
+        [Authorize]
+        public async Task<IActionResult> GetWizardInformation(string sourceUrl)
```

Comment:
This endpoint needs to be able to handle not just URL's but also other URI's. Our project model and front end wizard already support URI's. While this endpoint currently only retrieves projects from URL's (General URL's, GitLab and GitHub), this can quickly expand to other types of URI's (For example, books by ISBN codes).